src/commands/remove-demo.ts

```diff
@@ -0,0 +1,98 @@
+import { GluegunToolbox } from "gluegun"
+import { demo } from "../tools/demo"
+import type { CommentType } from "../tools/demo"
+import { createGetAllFilePaths } from "../tools/path"
+import { p, warning } from "../tools/pretty"
+
+module.exports = {
+  alias: ["rd", "remove-demos"],
+  description: "Remove demo code from generated boilerplate",
+  run: async (toolbox: GluegunToolbox) => {
+    const { parameters, patching, filesystem } = toolbox
+
+    const CWD = process.cwd()
+    const TARGET_DIR = parameters.first ?? CWD
+    p()
+    p(`Removing demo code from '${TARGET_DIR}'`)
+
+    const getAllFilePaths = createGetAllFilePaths(filesystem)
+    const paths = getAllFilePaths(TARGET_DIR)
+
+    // Go through every file path and handle the operation for each demo comment
+    const demoCommentResults = await Promise.allSettled(
+      paths.map(async (path) => {
```

Comment:
Decided to go with 
```ts
Promise.allSettled(
    paths.map(async (path) => {
        // ...
    })
)`
```
So that all of these will be processed in parallel instead of synchronously like `for of` or `.forEach`

Trade off is that we don't have a guarantee that they will be processed the same way each time, but I think that's a worth while trade off for a potential speed improvement

Reply:
If this causes issues, I'm okay if we change to synchronous. Speed isn't a _yuge_ deal here, since it's rare people will run this command very often.